python/tests/test_framework/test_configs/cb.json

```diff
@@ -49,90 +49,56 @@
-                                    7
-                                ]
-                            }
-                        ]
-                    },
-                    {
-                        "+": [
-                            {
-                                "--cover": [
-                                    1,
-                                    2,
-                                    3
-                                ]
-                            }
-                        ]
-                    },
-                    {
-                        "+": [
-                            {
-                                "--squarecb": [
-                                    "--gamma_scale 1000",
-                                    "--gamma_scale 10000"
-                                ]
-                            }
-                        ]
-                    },
-                    {
-                        "+": [
-                            {
-                                "--synthcover": [
-                                    ""
-                                ]
-                            }
-                        ]
-                    },
-                    {
-                        "+": [
-                            {
-                                "--regcb": [
-                                    ""
-                                ]
-                            }
-                        ]
-                    },
-                    {
-                        "+": [
-                            {
-                                "--softmax": [
-                                    ""
-                                ]
-                            }
-                        ]
-                    }
-                ]}
-            ]],
+            {
+                "--epsilon": [
+                    0.1,
+                    0.2,
+                    0.3
+                ]
+            },
+            {
+                "--first": [
+                    1,
+                    2
+                ]
+            },
+            {
+                "--bag": [
+                    5,
+                    6,
+                    7
+                ]
+            },
+            {
+                "--cover": [
+                    1,
+                    2,
+                    3
+                ]
+            },
+            {
+                "--squarecb": [
+                    "--gamma_scale 1000",
+                    "--gamma_scale 10000"
+                ]
+            },
+            {
+                "--synthcover": [
+                    ""
+                ]
+            },
+            {
+                "--regcb": [
+                    ""
+                ]
+            },
+            {
+                "--softmax": [
+                    ""
+                ]
+            }
+        ],
+        "grids_expression": "g0 * (g1 + g2 + g3 + g4 + g5 +g6 + g7 + g8)",
```

Comment:
It is hard to track down which one is g3 for example. Maybe we can turn current list into disctionary with explicit namings?